src/client/java/teammates/client/scripts/scalabilitytests/InstructorCourseEnrollPageScaleTestDataGenerator.java

```diff
@@ -0,0 +1,50 @@
+package teammates.client.scripts.scalabilitytests;
+
+import org.kohsuke.randname.RandomNameGenerator;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.HashSet;
+
+/**
+ * Generates test data for InstructorCourseEnrollPageScaleTest.
+ */
+public final class InstructorCourseEnrollPageScaleTestDataGenerator {
+
+    private static final String FILENAME = "InstructorCourseEnrollPageScaleTestData";
+
+    private static final String HEADER = "Section | Team | Name | Email | Comments\n";
+
+    private static final int[] studentGroups = {10, 20, 50, 75, 100, 150};
+
+    private InstructorCourseEnrollPageScaleTestDataGenerator() {
+    }
+
+    @SuppressWarnings("PMD.ConsecutiveLiteralAppends")
```

Comment:
Any reason for this? Line 35 could use `String.format` to look neat and avoid consecutive literal appends.